src/site/content/en/blog/eventing-deepdive/index.md

```diff
@@ -0,0 +1,572 @@
+"click on #B in capturing phase"
+"click on #C in capturing phase"
+"click on #C in bubbling phase"
+"click on #B in bubbling phase"
+"click on #A in bubbling phase"
+```
+
+You can interactively play with this in the live demo below. Click on the `#C` element in the live demo and observe the console output.
+
+<div style="height: 680px; width: 100%;">
+  <iframe src="https://silicon-brawny-cardinal.glitch.me/stop-propagation-capturing-C.html" loading="lazy" style="height: 100%; width: 100%; border: 0;"></iframe>
+</div>
+
+One more, just for fun. What happens if we call `stopPropagation()` in the _target phase_ for `#C`?
+Recall that the "target phase" is the name given to the period of time when the event is _at_ its
+target. It would result in the following output:
+
+```bash
+"click on document in capturing phase"
+"click on <html> in capturing phase"
+"click on <body> in capturing phase"
+"click on #A in capturing phase"
+"click on #B in capturing phase"
+"click on #C in capturing phase"
+```
+
+{% Aside %} It would be technically more accurate if I had logged "click on #C in target phase".
+However, I chose to use the term "capturing" and "bubbling" for both of `#C`'s event handlers to
+make it clear that handlers can be executed in both phases of the event's lifecycle. Just know that
+technically, the time when the event is at `#C` is officially known as the "target phase".
+{% endAside %}
+
+Note that the event handler for `#C` in which we log "click on #C in the capturing phase" _still_
+executes, but the one in which we log "click on #C in the bubbling phase" does not. This should make
+perfect sense. We called `stopPropagation()` _from_ the former, so that is the point at which the
+event's propagation will cease.
+
+You can interactively play with this in the live demo below. Click on the `#C` element in the live demo and observe the console output.
+
+<div style="height: 680px; width: 100%;">
+  <iframe src="https://silicon-brawny-cardinal.glitch.me/stop-propagation-bubbling-A.html" loading="lazy" style="height: 100%; width: 100%; border: 0;"></iframe>
+</div>
+
+In any of these live demos, I encourage you to play around. Try clicking on the `#A` element only or
+the `body` element only. Try to predict what will happen and then observe if you are correct. At
+this point, you should be able to predict pretty accurately.
+
+### `event.stopImmediatePropagation()`
+
+What is this strange, and not oft-used method? It's similar to stopPropagation, but rather than
+stopping an event from traveling to descendents (capturing) or ancestors (bubbling), this method
+only applies when you have more than one event handler wired up to a single element. Since
+`addEventListener()` supports a multicast style of eventing, it's completely possible to wire up an
+event handler to a single element more than once. When this happens, (in most browsers), event
+handlers are typically executed in the order they were wired up. Calling
```

Comment:
Typically implies it might happen some other way. Remove that word?

Reply:
Good point. Done.